api-gateway/src/main/java/com/example/apigateway/resources/ApiPaths.java

```diff
@@ -0,0 +1,31 @@
+package com.example.apigateway.resources;
+
+public class ApiPaths {
```

Comment:
Утилитные классы/классы с константами обычно делают final и с приватным дефолтным конструктором, чтобы от них нельзя было унаследовать и создать экземпляр класса.
```suggestion

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class ApiPaths {
```